packages/ripple-binary-codec/src/utils.ts

```diff
@@ -0,0 +1,149 @@
+// Don't know why this comes from NodeJS as it is valid in the browser
```

Comment:
nit: I prefer writing in an instructional tone rather than a 1st person one when committing. Conveys the same information without conveying the feeling of confusion since the confusion can be a bit distracting.
```suggestion
// Even though this comes from NodeJS it is valid in the browser
```